jwst/stpipe/resources/schemas/step_config-0.1.0.yaml

```diff
@@ -0,0 +1,23 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/yaml-schema/draft-01
+id: asdf://stsci.edu/stpipe/schemas/step_config-0.1.0
+title: Calibration pipeline Step config file
+description: >
+  File containing saved configuration for an stpipe Step
+  instance.
+
+type: object
+properties:
+  parameters:
+    type: object
+    properties:
+      class:
```

Comment:
`class` is a reserved word in Python and should not be used.  Though clearly if we change it, there will be a small amount of backwards incompatibility.

Reply:
Why is that relevant in an ASDF file?  This gets mapped to the string key "class" in a dict.

Reply:
Ah true.  When this was a datamodel, it meant that it had an attribute named `class` which caused problems in testing.  See #3874.

Carry on.